### DISCUSSION TITLE
Overlayfs implementation and ext4 SD card support

### DISCUSSION THREAD

**Discussion by archandanime** [gtxaspec/wz_mini_hacks#142] [Development]
How do you plan to implement overlayfs?
How do you think about adding ext4 support for SD card? (adding ext4 support means kernel partition needs to be flashed because uboot can't read factory_t31_ZMC6tiIDQN on ext4 SD card)

**Comment by gtxaspec** [comment]
regarding overlayfs, similar to how openwrt does it.  ext4 support for sdcard doesn't seem necessary, as we can use an ext4 image (like atomcam_tools does) to host all the files, storing the image on the fat32 card.  I'm not sure if its possible, but we could possibly also support sd card partitioning, a small fat32 partition, then a larger extfat partition as well.
